src/mod_vcard_ldap.erl

```diff
@@ -57,23 +57,22 @@
 -define(PROCNAME, ejabberd_mod_vcard_ldap).
 
 -record(state,
-        {serverhost = <<"">>        :: binary(),
-         myhost = <<"">>            :: binary(),
-         eldap_id                   :: {jid:lserver(), binary()},
-         base = <<"">>              :: binary(),
-         password = <<"">>          :: binary(),
+        {serverhost = <<>>          :: binary(),
+         myhost = <<>>              :: binary(),
+         eldap_id                   :: mongoose_utils:eldap_id(),
```

Comment:
Here it's `mongoose_utils:eldap_id()`, but in `src/auth/ejabberd_auth_ldap.erl` it is `eldap_utils:eldap_id()`.